doc/03-Configuration.md

```diff
@@ -24,6 +24,15 @@ database                 | **Required.** Database database.
 user                     | **Required.** Database username.
 password                 | **Required.** Database password.
 
+## Logging Configuration <a id="configuration-logging"></a>
+
+Configuration of the logging component used by Icinga DB.
+
+Option                   | Description
+-------------------------|-----------------------------------------------
+level                    | **Optional.** Logger default level (debug, info, warn, error, dpanic, panic or fatal). Usually set to `debug` by default.
+options                  | **Optional.** Child loggers, with `Icinga Db component` => `logging level` map.<br /> `Icinga Db component`: database, redis, heartbeat, high-availability, config-sync, history, runtime-updates, overdue-sync, dump-signals, delta. <br /> `logging level`: debug, info, warn, error, dpanic, panic or fatal.
```

Comment:
I would not write long text in tables, but create a separate section under logging and document the components with a small description of what they do there. The description for `options` could then be something like this:

> **Optional.** Map of component name to logging level in order to set a different logging level for each component instead of the default one. See [logging components](#logging-components) for details.